pkg/execution/computionadmission/README.md

```diff
@@ -0,0 +1,18 @@
+# Compution
+
+## Admission Controller
```

Comment:
This thing is Atlassian specific. Shall we move it to an `/atlassian/` subpackage?